_drafts/randy/2022-07-17-10-second-video.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: "How Can 10 Second of Video Take All Day?"
```

Comment:
"How Can 10 Seconds of Video Take All Day" or "How Can a 10 Second Video Take All Day"

Reply:
latter